region/tourian/main.json

```diff
@@ -3970,6 +3970,55 @@
                     "Run into the pirates with speedbooster to save health over shinesparking.",
                     "A speedy jump can reach the top platform."
                   ]
+                },
+                {
+                  "name": "G-Mode Morph PB Pirate Kill",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "h_canArtificialMorphSpringBallBombJump"
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 5 }}
+                  ],
+                  "note": "IBJ or spring ball bomb jump to avoid the wall jump. The first pirate takes 2 PBs, the next 3 can be killed with 3 total PBs if they are placed under the middle pirate."
+                },
+                {
+                  "name": "G-Mode Morph PB Pirate Hitbox",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "h_canArtificialMorphSpringBallBombJump",
```

Comment:
```
{"or": [
  "h_canArtificialMorphIBJ",
  "h_canArtificialMorphSpringBallBombJump"
]},
```

Reply:
if the pirate is still alive, the short ibj next to it is really annoying

Reply:
With bomb horizontally its fine.  Or just bomb jumping up extra high.